test/fhirtypes/ElementDefinition.constrainCardinality.test.ts

```diff
@@ -226,15 +227,17 @@ describe('ElementDefinition', () => {
       expect(clone).toEqual(category);
     });
 
-    it('should throw InvalidMaxOfSliceError when sliced element max is constrained less than any individual slice max', () => {
+    it('should log a warning when sliced element max is constrained less than any individual slice max', () => {
       const category = respRate.elements.find(e => e.id === 'Observation.category');
+      console.log(category);
       const fooSlice = category.addSlice('FooSlice');
       fooSlice.max = '2';
-      const clone = cloneDeep(category);
-      expect(() => {
-        category.constrainCardinality(1, '1');
-      }).toThrow(/max of slice FooSlice \(2\) > max of sliced element \(1\)\./);
-      expect(clone).toEqual(category);
+      category.constrainCardinality(1, '1');
+      expect(loggerSpy.getAllMessages('warn')).toContain(
+        'Max of element Observation.category is > max of slice FooSlice'
```

Comment:
If you add a colon to the string (as suggested above), better fix the test too!
```suggestion
        'Max of element Observation.category is > max of slice: FooSlice'
```